### DISCUSSION TITLE
Mergify on Github Enterprise

### DISCUSSION THREAD

**Discussion by foutoucour** [Mergifyio/mergify#3708] [Q&A]
Hello Les Toulousains ;)

I was checking Mergify and it is a bot I would love to see in our projects. But, there is often a but, we are host on a private GitHub enterprise server.

Would Mergify app or mergify-engine be compatible with Github Enterprise?

Thank you for your work!
Jordi

**Comment by jd** [comment]
Yes, it does support GHES. Reach out to me at jd@mergify.com and we'll chat about it. 👍
